sample/SampleHost/Properties/PublishProfiles/brettsam-webjob - Web Deploy.pubxml

```diff
@@ -0,0 +1,22 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
(If you don't mind comments on your pr) likely you don't want this file to go in